src/coreclr/src/jit/importer.cpp

```diff
@@ -1440,6 +1440,19 @@ GenTree* Compiler::impAssignStructPtr(GenTree*             destAddr,
     }
     else if (compDoOldStructRetyping())
     {
+        if (dest->OperIs(GT_LCL_VAR) &&
```

Comment:
Git has missed this merge conflict, I changed the condition from:
```
else
{
  dest->gtType = asgType;
}
```
to
```
else if (compDoOldStructRetyping())
{
  dest->gtType = asgType;
}
```
and you added code under the original `else // no condition` that should not be under the change condition.

Could you please move `if (compDoOldStructRetyping())` to `dest->gtType = asgType;`?